modules/integration/src/test/resources/config/cli-config.yaml

```diff
@@ -0,0 +1,5 @@
+!!org.wso2.broker.client.resources.Configuration
```

Comment:
Can we remove this line from the config? Feels like this is something the users don't want to see.

Reply:
Are we exposing this config to the outside?

Reply:
I will do that remove that. I didn't worry about this because the user will not have anything to do with this configuration file. Read and Write this file will only be done by the CLI app